targets/csharp/source/source/Extensions/Models/PlayFabCloudScriptModelsExtensions.cs.ejs

```diff
@@ -1,13 +1,20 @@
+#if NETCOREAPP2_0
```

Comment:
It's possible this isn't the right #if condition.
I'm compiling a .NetCoreApp 2.0, but it's grey-ed out this entire file.